DOCUMENTATION.md

```diff
@@ -19,12 +19,17 @@
 * [3. Configuration](#3-configuration)
   * [3.1 Client Properties](#31-client-properties)
 * [4. Serialization](#4-serialization)
-  * [4.1. IdentifiedDataSerializable Serialization](#41-identifieddataserializable-serialization)
-  * [4.2. Portable Serialization](#42-portable-serialization)
-    * [4.2.1. Versioning for Portable Serialization](#421-versioning-for-portable-serialization)
-  * [4.3. Custom Serialization](#43-custom-serialization)
-  * [4.4. Global Serialization](#44-global-serialization)
-  * [4.5. JSON Serialization](#45-json-serialization)
+  * [4.1. Compact Serialization](#41-compact-serialization)
+    * [4.1.1. Schema Evolution](#411-schema-evolution)
+    * [4.1.2. Generic Record Representation](#412-generic-record-representation)
+    * [4.1.3. Compact Configuration and Usage](#413-compact-configuration-and-usage)
+    * [4.1.4. Limitations](#413-compact-configuration-and-usage)
```

Comment:
Link doesn't match to the text.

Reply:
Actually the markdown link checker now can check these links. I just need to merge upstream master into this branch.

Reply:
Ah sorry it won't work because the destination exists but a wrong one